src/OSPSuite.Core/Serialization/Xml/SerializationContext.cs

```diff
@@ -87,6 +87,10 @@ public DataColumnCollectionReference AddDataCollectionReference(ICache<Auxiliary
 
       public IDimension DimensionByName(string dimensionName)
       {
+         var found = DimensionFactory.TryGetDimension(dimensionName, out var dimension);
```

Comment:
This is the key difference here. Instead of throwing the exception if not found, we try to get it first